embed/etcd.go

```diff
@@ -59,6 +59,9 @@ type Etcd struct {
 }
 
 // StartEtcd launches the etcd server and HTTP handlers for client/server communication.
+// You'll probably want to wait on the .Server.ReadyNotify() channel that is
```

Comment:
`You` is bad style for documentation (https://github.com/coreos/docs/blob/master/STYLE.md#eschew-you). Just `The returned `Etcd.Server` is not guaranteed to have joined the cluster; wait on ReadyNotify() for the server to join.`